### ISSUE TITLE
Cannot call API inside remoteOptimizedImages.js

### ISSUE THREAD

**Issue by thuatz** [Niels-IO/next-image-export-optimizer#165]
## Context:
I have a image storage in the clould, it provide an API to fetch the list of images.

I use that list to download remote images for optimization.

### The following works fine in v1.7
**remoteOptimizedImages.js**

```
module.exports = new Promise(async (resolve) => {
  let images = [];
  try {
    images = await fetchAPI("/upload/files");
  } catch (error) {
    console.error(error);
  }
  resolve(images);
}
```
### Not working in v1.8
When I see **getRemoteImageURLs.js**, I found the changes:
```
    if (fs.existsSync(remoteImagesFilePath)) {
        const { default: remoteOptimizedImages } = await Promise.resolve(`${remoteImagesFilePath}`).then(s => __importStar(require(s)));
        remoteImageURLs = remoteOptimizedImages;
    }
```
compares to:
```
    if (fs.existsSync(remoteImagesFilePath)) {
        remoteImageURLs = await Promise.resolve(require(remoteImagesFilePath));
    }
```
(v1.7)

Could you please fix it or tell me the correct way to call APIs to get the list of image URLs?

**Comment by Niels-IO** [comment]
Hi, I addressed the issue in #167. When the new version is released, you could then use your API call like this:
module.exports = fetchAPI("/upload/files").catch((error) => {
  console.error(error);
  return [];  // return an empty array in case of error
});

**Comment by Niels-IO** [comment]
Please test the solution in version 1.8.3
